js/atree.js

```diff
@@ -1389,9 +1397,22 @@ const atreeNodeAtlasPositions = {
     "node_shaman": 8
 }
 const atreeNodeTileSize = 32;
-const atreeNodeAtlasImg = make_elem("img", [], {src: "../media/atree/icons.png"});
+const atreeNodeAtlasImg = make_elem("img", [], {src: "../media/atree/icons.png", loaded: false});
+atreeNodeAtlasImg.addEventListener("load", () => {
+    atreeNodeAtlasImg.loaded = true;
+    let drawList = atlasToDraw;
+    atlasToDraw = [];
+    for (const toDraw in drawList) {
+        drawAtlasImage(toDraw[0], toDraw[1], toDraw[2], toDraw[3]);
+    }
+});
 
+var atlasToDraw = [];
 function drawAtlasImage(canvas, img, pos, tileSize) {
+    if (!img.loaded) {
+        atlasToDraw.push([canvas, img, pos, tileSize]);
```

Comment:
possibly, separate out the two lists (maybe a `Map[img, List[draw_cmd]]`)? though idk if it really matters